PKGBUILD

```diff
@@ -0,0 +1,17 @@
+pkgname=alfis
+pkgver=0.3.12
+pkgrel=1
+pkgdesc="Alternative Free Identity System"
+arch=('x86_64')
+license=('AGPL3')
+url='https://github.com/Revertron/Alfis'
+depends=('webkit2gtk')
+backup=("etc/$pkgname.toml")
```

Comment:
Change `.toml` to `.conf` please. We will use that extension like Yggdrasil does :)

Reply:
Done.